### PR TITLE
new ICDS inventory

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team 

- [ ] add riak IP addresses
- [x] configure postgres vars
- [ ] remote_couch_backup var - is this needed?

### CODE REVIEW

File: fab/environments.yml [hunk 1]
```diff
@@ -378,6 +377,163 @@ icds:
       XFormToElasticsearchPillow:
         num_processes: 4
 
+icds-new:
+  environment: icds
```
Comment (sravfeyn) [inline fab/environments.yml]: Should we name it different (e.g. `icds2`) so it doesn't conflict with `icds` if we deploy from local machine?

File: fab/inventory/icds-new [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+10.247.164.47
+
+[formplayer0:vars]
+hostname='formplayer0'
+
+[patch0]
+10.247.63.132
+
+[proxy:children]
+web0
```
Comment (sravfeyn) [inline fab/inventory/icds-new:483]: Are we not having standalone proxy?

Comment (snopoke) [reply fab/inventory/icds-new:483]: We can but I don't think it's necessary

File: fab/environments.yml (hunk 1 above)
Comment (snopoke) [reply fab/environments.yml]: yea, this should be `icds-new`

**Pull request merged**